apps/site/src/app/schedule/page.tsx

```diff
@@ -1,16 +1,29 @@
+import { PortableText } from "@portabletext/react";
+
 import ShiftingCountdown from "./components/ShiftingCountdown/ShiftingCountdown";
+import { getSchedule } from "./components/getSchedule";
+import SchedulePage from "./sections/SchedulePage";
 
 export const revalidate = 60;
 
```

Comment:
Issue: Could we export metadata like how we do for the login `page.tsx`? We should have a descriptive title for our pages (such as `Schedule | IrvineHacks 2024`).